metricflow-semantics/metricflow_semantics/test_helpers/semantic_manifest_yamls/simple_manifest/metrics.yaml

```diff
@@ -783,22 +783,3 @@ metric:
       name: listings
   filter: "{{ Metric('views', ['listing']) }} > 10"
   time_granularity: week
----
-metric:
-  name: bookings_before_dec_20_2019
-  description: Bookings up to the end of 2022
-  type: simple
-  type_params:
-    measure: bookings
-  filter: "{{ TimeDimension('metric_time') }} < '2012-12-20'"
```

Comment:
Oh, I think I know why this is. Timestamp literals in Trino have to take the form `TIMESTAMP <literal>` so we'd need to do some substitution somewhere.

If the error is due to the filter it means we have to do custom rendering against the filter expr.

Reply:
Yep. I couldn't come up with an easy way to fix it here and wasn't sure it was worth doing the hard fix for an engine we barely use.

Reply:
Agreed. I think the right way to deal with this is via a more holistic approach to filter expression inputs. In the meantime, having a small gap in Trino test coverage - particularly one where the main difference essentially boils down to how we go about pasting in the user-provided expression at render time - seems fine to me.